### ISSUE TITLE
Start tracking change in NEWS.md

### ISSUE THREAD

**Issue by maelle** [lvaudor/glitter#40]
@lvaudor ça te va si nous utilisons https://github.com/cynkra/fledge#fledge?

**Comment by lvaudor** [comment]
yes, ça me va... Même moi ça me servira de savoir où on en est huhu

**Comment by maelle** [comment]
Pour clarifier tu préfères

- Un NEWS.md mis à jour à la main
- Utiliser fledge https://cynkra.github.io/fledge/ Cela demande d'utiliser des "-" dans les messages de commit pour indiquer les nouvelles à lister.
